src/com/bosiy/Triangle.java

```diff
@@ -0,0 +1,31 @@
+package com.bosiy;
+
+public class Triangle {
+    private final int sideA;
+    private final int sideB;
+    private final int sideC;
+
+    public Triangle(int sideA, int sideB, int sideC) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.sideC = sideC;
+    }
+
+    @Override
+    public String toString() {
+        return "side a = " + sideA + "; " + " side b = " + sideB + "; " +
+                " side c = " + sideC + "; ";
+    }
+
+    public String variation() {
```

Comment:
rename method to verb.
all methods should be named as verbs.

for ex. getVariation, getTriangleType, getType etc.

Reply:
fixed